File: Python/Samples/Template/UtTemplate.py
import unittest
from ProductFixingTemplate import ProductFixingTemplate
from TrfFixing import TrfFixing
from DkoFixing import DkoFixing

class UtTemplate(unittest.TestCase):

    def test_template(self):
        trfFixing = TrfFixing()
        trfFixing.fixing()

        dkoFixing = DkoFixing()
        dkoFixing.fixing()

        self.assertTrue(True)


if __name__ == '__main__':
    unittest.main()

File: CSharp/DP.UnitTest/UtTemplate.cs
using System;
using System.Diagnostics;
using DP.Domain.Samples.Template;
using DP.UnitTest.Utility;
using Xunit;
using Xunit.Abstractions;

namespace DP.UnitTest
{
    public class UtTemplate
    {
        private readonly ITestOutputHelper _output;

        public UtTemplate(ITestOutputHelper output)
        {
            this._output = output;
            System.Diagnostics.Trace.Listeners.Add(new XunitTraceListener(this._output)); 
        }

        [Fact] 
        public void TestTemplate()
        {
            var trfFixing = new TrfFixing();
            trfFixing.Fixing();

            var dkoFixing = new DkoFixing();
            dkoFixing.Fixing();

            Assert.True(true);
        }
    }
}
